Source/carthage/Extensions.swift

```diff
@@ -31,26 +30,26 @@ private let outputQueue = { () -> dispatch_queue_t in
 /// A thread-safe version of Swift's standard println().
 internal func println() {
```

Comment:
If we have the two funcs
- `internal func print(terminator terminator: String = "\n")`
- `internal func print<T>(object: T, terminator: String = "\n")`

the former is used for the following:

```
carthage.print(formatting.bullets + "Found " + formatting.path(string: path))
```

What should we do here?